_episodes/00-oceanhack-intro.md

```diff
@@ -66,18 +78,12 @@ Be sure to arrive at Oceanhackweek with your own [GitHub](https://github.com/) a
     sudo apt install git-all
   ```
 
-### Getting setup with JupyterHub on AWS
-
-For those who have joined the oceanhackweek [participants_2018 team](https://github.com/orgs/oceanhackweek/teams/participants_2018/members) in github, you can try accessing the Jupyterhub server we will be using throughout the program by clicking [here](#). Make sure you are able to select the Python 3 kernel on the top right corner when you click on the "New" button. You will be logging into Jupyterhub using your Github credentials.
-
-### Brushing up on Python
+### Getting started with Bash
 
-Given all the heavy use of python during the oceanhackweek, we will not be able to provide instruction in Python fundamentals. Here are some excellent resources we recommend using to brush up on your Python skills in advance of our event:
+During the hackweek it will be useful to know how to navigate between files from the command line. If you are not familiar with the unix shell commands you can review the first free sections of this [Software Carpentry Shell Novice](https://swcarpentry.github.io/shell-novice/) lesson. On Windows, use the Git Bash terminal to run these commands.
 
-* [Software carpentry lessons](https://software-carpentry.org/lessons/), especially the [Python tutorial](http://swcarpentry.github.io/python-novice-inflammation/)
-* [Python Data Science Handbook](https://github.com/jakevdp/PythonDataScienceHandbook)
 
-### Getting started with Bash
+### Command line editor
 
-During the hackweek it will be useful to know how to navigate between files from the command line. If you are not familiar with the unix shell commands you can review the first free sections of this [Software Carpentry Shell Novice](https://swcarpentry.github.io/shell-novice/) lesson. On Windows, use the Git Bash terminal to run these commands. 
+When working on the command line, it is often handy to modify file content directly from there. For that you can use a command line editor such as `nano`. On Mac and Linux it is usually preinstalled, but for Windows you can follow the instructions in this [link](http://carpentries.github.io/workshop-template/#editor) to set it up.
```

Comment:
Do they really need a command line editor? Not sure what you're thinking in terms of usage of this during the hackweek. Thanks.

Reply:
I will use it during the git tutorial. I will give them an option to use nano as I do, or they can use their favorite editor. I think it will be easier if they follow with nano, as they can stay in one place (otherwise they may get confused between terminal commands vs file editing).